qutip/solver/parallel.py

```diff
@@ -385,9 +385,12 @@ def loky_pmap(task, values, task_args=None, task_kwargs=None,
 
 
 _get_map = {
+    parallel_map: parallel_map,
```

Comment:
I would prefer not to do this -- it's a mess. The string keys make such arguments simple and, e.g., serializable, and including functions as keys breaks that.

I'd be much happier to just ask people to change their arguments to strings in v5.